app/src/main/java/org/permanent/permanent/ui/activities/MainActivity.kt

```diff
@@ -121,6 +127,12 @@ class MainActivity : PermanentBaseActivity(), Toolbar.OnMenuItemClickListener {
         }
     }
 
+    // On settings icon click
+    override fun onMenuItemClick(menuItem: MenuItem?): Boolean {
```

Comment:
do we still need this?

Reply:
Yes for when we will also use the right drawer menu.